benefits/static/css/styles.css

```diff
@@ -787,6 +787,7 @@ a.card:focus-visible {
 .landing .box {
   background: var(--landing-box-background);
   border-top: var(--landing-box-border);
+  border-radius: calc(8rem / 16);
```

Comment:
Not a blocker, but I just noticed we have a few different ways / places of specifying `border-radius`, I wonder if we want to consolidate at some point:

* Root variable [`--border-radius: calc(3rem / 16)`](https://github.com/cal-itp/benefits/blob/fix/1578-rounded-corners/benefits/static/css/styles.css#L38), used for [`.signout-link`](https://github.com/cal-itp/benefits/blob/fix/1578-rounded-corners/benefits/static/css/styles.css#L432) and [`.form-container .form-control`](https://github.com/cal-itp/benefits/blob/fix/1578-rounded-corners/benefits/static/css/styles.css#L499)
* Direct [8px rule on `.card`](https://github.com/cal-itp/benefits/blob/fix/1578-rounded-corners/benefits/static/css/styles.css#L640)
* Root variable [`--modal-border-radius: 8px;`](https://github.com/cal-itp/benefits/blob/fix/1578-rounded-corners/benefits/static/css/styles.css#L666), overridden for wider screens to [`--modal-border-radius: 4px;`](https://github.com/cal-itp/benefits/blob/fix/1578-rounded-corners/benefits/static/css/styles.css#L672), used for [`.modal-content`](https://github.com/cal-itp/benefits/blob/fix/1578-rounded-corners/benefits/static/css/styles.css#L690)

And now the one from this PR.

Reply:
Yeah I noticed this too. We can refactor at another time.